src/app/Fake.BuildServer.TeamCity/TeamCity.fs

```diff
@@ -361,9 +369,8 @@ module TeamCity =
             with get() = RecentlyFailedTests.cache.Value
 
     /// Implements a TraceListener for TeamCity build servers.
-    /// ## Parameters
-    ///  - `importantMessagesToStdErr` - Defines whether to trace important messages to StdErr.
-    ///  - `colorMap` - A function which maps TracePriorities to ConsoleColors.
+    /// 
+    /// See [the documentation](https://confluence.jetbrains.com/display/TCD18/Build+Script+Interaction+with+TeamCity) for more information
```

Comment:
Documentation needs to go to the `TeamCity` module above, I don't think `internal` types are part of the API reference docs. (I'd recommend copying the section from TeamFoundation to have a starting point)

Reply:
Ok, I'll update tomorrow